master/reference/cni-plugin/configuration.md

```diff
@@ -157,16 +157,19 @@ As a convenience, the API location location can also be configured directly, e.g
 ### Enabling Kubernetes Policy
 
 If you wish to use the Kubernetes NetworkPolicy API then you must set a policy type in the network config.
-There is a single supported policy type, `k8s` which uses the Kubernetes NetworkPolicy API in conjunction with the `calico/kube-policy-controller`.
+There is a single supported policy type, `k8s` which uses the Kubernetes NetworkPolicy API. When set,
```

Comment:
If we are in agreement that `NetworkPolicy` is a resource of a larger API then I would suggest "NetworkPolicy API" --> "`NetworkPolicy` resource" (notice here that I have enclosed NetworkPolicy in backticks)

Reply:
It's both an API and a resource within a larger API.  I've seen it called the Kubernetes NetworkPolicy API as well as the Kubernetes NetworkPolicy resource.